usecases/schema/executor.go

```diff
@@ -176,27 +174,21 @@ func (e *executor) UpdateTenants(class string, req *api.UpdateTenantsRequest) er
 		})
 	}
 
-	commit, err := e.migrator.UpdateTenants(ctx, cls, updates)
-	if err != nil {
+	if err := e.migrator.UpdateTenants(ctx, cls, updates); err != nil {
 		e.logger.WithField("action", "update_tenants").
 			WithField("class", class).Error(err)
 		return err
 	}
-
-	commit(true) // commit update of tenants
 	return nil
 }
 
 func (e *executor) DeleteTenants(class string, req *api.DeleteTenantsRequest) error {
 	ctx := context.Background()
-	commit, err := e.migrator.DeleteTenants(ctx, class, req.Tenants)
-	if err != nil {
+	if err := e.migrator.DeleteTenants(ctx, class, req.Tenants); err != nil {
 		e.logger.WithField("action", "delete_tenants").
 			WithField("class", class).Error(err)
+		// TODO no return err?
```

Comment:
**suggestion (if-minor):** do we want to address it in this PR to avoid the TODO ?

Reply:
Apparently error is not expected here. Executor is tested not to return error even if migrator does return.

Reply:
interesting, I am curious why we don't expose the error on delete 🤔 , cc @redouan-rhazouani

Reply:
Here, the shard is already deleted in the schema. If something happens, this would be an internal issue that the server needs to rectify. The worst-case scenario is that we will have a dangling shard